### DISCUSSION TITLE
Adding GitHub Releases as a mirror for doxygen

### DISCUSSION THREAD

**Discussion by ok-coder1** [Homebrew/discussions#5222] [Tap maintenance and Homebrew development] [answered]
### Description

I think it will be OK to add GitHub's releases page's source code file (.tar.gz) as a mirror to `doxygen`.

**Comment by SMillerDev** [comment, marked as answer]
Go for it!

**Reply by ok-coder1** [reply]
Should I create a PR?

**Reply by SMillerDev** [reply]
Yes, see https://docs.brew.sh/Updating-Software-in-Homebrew for guidance

**Comment by ok-coder1** [comment]
Thanks!
